src/Packlist/PacklistForm.js

```diff
@@ -1,43 +1,55 @@
-import React, { useState, useRef } from 'react'
+import React, { useState, useRef, useEffect } from 'react'
 import { useForm } from 'react-hook-form'
 import { v4 as uuid } from 'uuid'
 import styled from 'styled-components/macro'
 import Input from '../common/Input'
 import Label from '../common/Label'
 import Checkbox from '../common/Checkbox'
 import ErrorMessage from '../common/ErrorMessage'
-import PropTypes from 'prop-types'
 import ListContainer from '../common/ListContainer'
 import ListItem from '../common/ListItem'
 import usePacklists from './usePacklists'
 import reloadIcon from '../img/reloadIcon.svg'
 import saveIcon from '../img/saveIcon.svg'
 import addIcon from '../img/addIcon.svg'
+import { useParams, useHistory } from 'react-router-dom'
+import usePacklistForm from './usePacklistForm'
 
-PacklistForm.propTypes = {
-  onPacklistSave: PropTypes.func.isRequired,
-}
-
-export default function PacklistForm({ onPacklistSave }) {
+export default function PacklistForm() {
   const { register, handleSubmit, reset, errors } = useForm()
-  const [items, setItems] = useState([])
+  const { editPacklistID } = useParams()
```

Comment:
The name "editPacklistID" may be misleading. It sounds like a function that edits a packlist ID. Maybe you can call it something like "currentPacklistId" or "editablePacklistId"

Reply:
I think about this, thanks